src/Tribe/PUE/Checker.php

```diff
@@ -1890,12 +1945,12 @@ public function check_for_api_key_error( $value, $field_id, $validated_field ) {
 		 * @see plugins_api()
 		 *
 		 * @param mixed        $result The result object or NULL.
-		 * @param string       $action The type of information being requested from the Plugin Install API.
+		 * @param string|null  $action The type of information being requested from the Plugin Install API.
 		 * @param array|object $args   Arguments used to query for installer pages from the Plugin Install API.
 		 *
 		 * @return mixed
 		 */
-		public function inject_info( $result, $action = null, $args = null ) {
+		public function inject_info( $result, string $action = null, $args = null ) {
```

Comment:
Just saw this... Should we not make the second param nullable given it can be either?
```suggestion
		public function inject_info( $result, ?string $action = null, $args = null ) {
```